### ISSUE TITLE
Can't update multi-user fields on v3

### ISSUE THREAD

**Issue by DA-Mena** [pnp/pnpjs#2265]
### Category
- [ ] Enhancement
- [*] Bug
- [ ] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [3.2.0 ]

Please specify what version(s) of SharePoint you are targeting: [Online]

### Expected / Desired Behavior / Question

This should've worked:
`await sp.web.lists.getById(listId).items.getById(itemId).update({ mutliuserId: { results: [entityId] }}, "*")`

But reading around I found list item entity was dropped because of http header content-type is not in verbose.

### Observed Behavior

I got the error "An unexpected 'StartObject' node was found when reading from the JSON reader. A 'StartArray' node was expected multi user field"

### Steps to Reproduce

Use PnP Js v3 to update a multiuser field from an old codebase that used the format in pnpjs v2

**Comment by patrick-rodgers** [comment]
Hi @DA-Mena - there was a [docs bug](https://pnp.github.io/pnpjs/sp/items/#user-fields) early in v3 that should be resolved. You no longer need the "results" part so your request should work like this:

```TS
await sp.web.lists.getById(listId).items.getById(itemId).update({ mutliuserId: [entityId] }, "*");
```

This is because we are no longer using verbose metadata by default, which required the results structure. Please give that a try and let us know if things are back to working for you.

**Comment by juliemturner** [comment]
After reviewing the docs are updated and correct for adding/updating multi-user field... that said I did add a few things to make it even more clear that the correct payloads for updating fields is the same as for adding fields. https://pnp.github.io/pnpjs/sp/items/#user-fields

**Comment by patrick-rodgers** [comment]
Going to close this as resolved as we included the docs updates in the 3.3.2 patch fix.
